apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -32,20 +32,20 @@ class Starter_Page_Templates {
 	private function __construct() {
 		$this->templates_cache_key = implode(
 			'_',
-			[
+			array(
 				'starter_page_templates',
 				PLUGIN_VERSION,
 				get_option( 'site_vertical', 'default' ),
 				get_locale(),
-			]
```

Comment:
This is odd; `[` is newer and we should support it. Are you sure you have the right version of `phpcs` installed?

I was surprised config doesn't define version here: https://github.com/Automattic/wp-calypso/blob/e9d236f7c95333fc09928cbd8a764b66c400eb92/apps/phpcs.xml

Not entirely sure what it should be or how it should be defined ([had a look at Jetpack](https://github.com/Automattic/jetpack/blob/16ec81984f5b14efe89dae5b36acf1e5c740b125/.phpcs.xml.dist#L4))

@vindl do you know what's going on here?

Reply:
Kind of out of my depth here, but `$ ./vendor/bin/phpcs --version` says `3.5.3`. When I first commited the pre-commit hook complained at me so I ran `composer install` at the root of my `wp-calypso` repo.

Reply:
There's some documentation at PCYsg-ly5-p2 (could just be moved to FSE repo as public docs...).

Reply:
Yeah, phpcs keeps auto-formatting these files to the old array syntax. I'm not sure why; I don't think anyone has had a chance to look into it

Reply:
I wonder if defining new enough PHP version like in https://github.com/Automattic/jetpack/blob/997026990b6d74f843ce7c5137cd2e7e8672ab2b/.phpcs.xml.dist#L4 would help.

Reply:
I tried setting the `testVersion` but alas that didn't fix it. I also pinned all the dependencies to the same ones used by Jetpack, but that also wasn't it.

Not a long term solution, but I'm just going to commit again and skip hooks so that the diff is smaller.

Reply:
I was wondering if we need to switch to `NeutronStandard` which includes the [DisallowLongformArraySniff](https://github.com/Automattic/phpcs-neutron-standard/blob/master/NeutronStandard/Sniffs/Arrays/DisallowLongformArraySniff.php)?

Reply:
(But skipping the hooks to get your diff smaller for now sounds like the way to go 👍)

Reply:
skipping the hooks to ~get your diff smaller for now~ punt the problem to the next person 😅